main/src/ca/uwaterloo/flix/tools/pkg/ReleaseError.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2024 Holger Dal Mogensen
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ca.uwaterloo.flix.tools.pkg
+
+import ca.uwaterloo.flix.tools.pkg.github.GitHub
+import ca.uwaterloo.flix.util.Formatter
+
+sealed trait ReleaseError {
+  /**
+    * Returns a human-readable and formatted string representation of this error.
+    */
+  def message(f: Formatter): String
+}
+
+object ReleaseError {
+  case object MissingManifest extends ReleaseError {
+    override def message(f: Formatter): String =
+      s"""
+         | Cannot create a release without a `flix.toml` file.
```

Comment:
I am puzzled by the one extra space. I saw it was already used in the other error messages, but it was not intended. 

I guess we can fix it in a separate PR to keep things focused.